gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/grpc.py.j2

```diff
@@ -55,8 +55,8 @@ _LOGGER = std_logging.getLogger(__name__)
 
 class _LoggingClientInterceptor(grpc.UnaryUnaryClientInterceptor):  # pragma: NO COVER
     def intercept_unary_unary(self, continuation, client_call_details, request):
-        request_metadata = client_call_details.metadata
-        if CLIENT_LOGGING_SUPPORTED and _LOGGER.isEnabledFor(std_logging.DEBUG):
+        if CLIENT_LOGGING_SUPPORTED and not _LOGGER.isEnabledFor(std_logging.NOTSET):  # pragma: NO COVER
```

Comment:
We should either revert these checks in this function to be the explicit checks for `DEBUG`, since we know that's what we're logging, or we should apply here the fix in my comment at the third logging-check location in this file, outside this function.

Reply:
> the third logging-check location in this file, outside this function

Is that the guard for the logging channel? I just removed it based on the other comment.